resources/application-connector/values.yaml

```diff
@@ -38,11 +38,11 @@ global:
     dir:
     version: fbf1000b
   event_service:
-    dir: develop/
-    version: e029fcf4
+    dir: 
```

Comment:
do you think we still need this key? should we remove it?

Reply:
it is still referenced in the templates, so we cannot just remove it from the values

Reply:
Yes, I understand, would it be cleaner if we removed it from the templates as well?